src/core/WallGraphicsScene.cpp

```diff
@@ -80,24 +79,23 @@ void WallGraphicsScene::drawBackground( QPainter* painter, const QRectF& rect )
         return;
     }
 
-    painter->beginNativePainting();
```

Comment:
I would keep them as it ensures to be always in a consistent OpenGL state along different Qt versions, according to the doc. For the things we draw it probably does not make a difference, but for instance a different active texture unit (which is resetted by beginNativePainting()) could hurt performance.